app/Console/Commands/ImportNasspGroupsCommand.php

```diff
@@ -0,0 +1,133 @@
+use Rogue\Models\Group;
+use Rogue\Models\GroupType;
+use Illuminate\Console\Command;
+
+class ImportNasspGroupsCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:nassp-groups-import {path}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create National Association of Secondary School Principals group types and groups from a CSV.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Returns a trimmed CSV value, or null if equal to the string "NULL".
+     *
+     * @param string
+     * @return string
+     */
+    public function sanitize($value)
+    {
+        $result = trim($value);
+
+        return $result == 'NULL' ? null : $value;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Sample CSV path: https://gist.githubusercontent.com/aaronschachter/a407e9882ebe01520644cc9c257c8ec9/raw/8c837c52cfe22866ba4b49f2cf2f12336ca297d8/groups.csv
+
+        $path = $this->argument('path');
+
+        info('rogue:nassp-groups-import: Loading csv from ' . $path);
+
+        // Make a local copy of the CSV.
+        $temp = tempnam(sys_get_temp_dir(), 'command_csv');
+
+        file_put_contents($temp, fopen($this->argument('path'), 'r'));
+
+        // Load CSV contents.
+        $csv = Reader::createFromPath($temp, 'r');
+
+        $csv->setHeaderOffset(0);
+
+        $numImported = 0;
+        $numFailed = 0;
+        $numSkipped = 0;
+
+        $nhsGroupType = GroupType::firstOrCreate([
+            'name' => 'National Honor Society',
+        ]);
+        $njhsGroupType = GroupType::firstOrCreate([
+            'name' => 'National Junior Honor Society',
+        ]);
+        $nascGroupType = GroupType::firstOrCreate([
+            'name' => 'National Student Council',
+        ]);
+
+        info('rogue:nassp-groups-import: Beginning import...');
+
+        /**
+         * Without specifying these column names, this command fails with error:
+         * "The header record must be an empty or a flat array with unique string value."
+         */
+        $records = $csv->getRecords(['LABEL NAME', 'NHS', 'NJHS', 'NASC', 'NASSP', 'ADDRESS1', 'ADDRESS2', 'CITY', 'STATE', 'POSTAL', 'COUNTRY', 'NCES']);
+
+        foreach ($records as $record) {
+            $name = trim($record['LABEL NAME']);
+
+            if (trim($record['NHS']) == 'NHS') {
+                $groupTypeId = $nhsGroupType->id;
+            } elseif (trim($record['NJHS']) == 'NJHS') {
+                $groupTypeId = $njhsGroupType->id;
+            } elseif (trim($record['NASC']) == 'NASC') {
+                $groupTypeId = $nascGroupType->id;
+            } else {
```

Comment:
The bottom 2 rows in this screenshot are an example of a school that appears twice in the CSV -- the first row will create it as a group within the NASC group type (National Student Council), and the second row will create it as a group within the NHS group type (National Honors Society). Both of these groups will share the same `external_id`, 480735000001 (these columns not visible in screenshot)

<img width="916" alt="Screen Shot 2020-06-25 at 11 02 05 AM" src="https://user-images.githubusercontent.com/1236811/85774257-842b3400-b6d3-11ea-88bd-77b6d21f4bb6.png">